tests/unit/test_snap.py

```diff
@@ -661,3 +661,21 @@ def test_install_local_args(self, mock_subprocess):
                 universal_newlines=True,
             )
             mock_subprocess.reset_mock()
+
+    @patch("charms.operator_libs_linux.v2.snap.subprocess.check_output")
+    def test_alias(self, mock_subprocess):
+        mock_subprocess.return_value = 0
```

Comment:
Shouldn't the return value of `check_output` be the output string? Probably `''` in this case.